include/attribute.h

```diff
@@ -33,241 +31,245 @@ struct BagWeapon
+    /*0x*/ u8 numPhysicalModifyBook;
+    /*0x*/ u8 numDistributionDevice;
+    /*0x*/ u8 numDestroySwitch;
+    /*0x*/ u8 numParaPowder;
+    /*0x*/ u8 numPoisonPowder;
+    /*0x*/ u8 numSleepPowder;
+    /*0x*/ u8 numRustPowder;
+    /*0x*/ u8 numForceLicense;
+    /*0x*/ u8 numDefendLicense;
+    /*0x*/ u8 numSuperManLicense;
+    /*0x*/ u8 numLegendWound;
+    /*0x*/ u8 numMindEyeGlasses;
+    /*0x*/ u8 numMartialTutorBook;
+    /*0x*/ u8 numStrongLuckAmulet;
+    /*0x*/ u8 numHealthBucket;
+    /*0x*/ u8 numLegendBamboo;
+    /*0x*/ u8 numOldGrindStone;
+    /*0x*/ u8 numRepareParts;
+    /*0x*/ u8 numLegendIronString;
+    /*0x*/ u8 numDareDevelLoveLetter;
+    /*0x*/ u8 numSuperMotor;
+    /*0x*/ u16 filler4DE;
+    /*0x*/ u8 suitRings[0xB]; // TODO: list name. 
+    /*0x*/ u8 suitBracelets[0xB];
+    /*0x*/ u8 suitShoes[0xB];
+    /*0x*/ u8 numMilitantShoe;
+    /*0x*/ u8 numEscapeShoe;
+    /*0x*/ u8 numNormalExtendRing;
+    /*0x*/ u8 numSuperExtendRing;
+    /*0x*/ u8 numUltraExtendRing;
+    /*0x*/ u8 numSpeedBracelet;
+    /*0x*/ u8 numGuardBracelet;
+    /*0x*/ u8 numAttackBracelet;
+    /*0x*/ u8 numCritEnhanceRing;
+    /*0x*/ u8 numMagBracelet;
+    /*0x*/ u8 numActiveRing;
+    /*0x*/ u8 numAwakeRing;
+    /*0x*/ u8 numDetoxifyRing;
+    /*0x*/ u8 numGoldenRing;
+    /*0x*/ u8 numBossRing1;
+    /*0x*/ u8 numBossRing2;
+    /*0x*/ u8 numCrafterRing;
+    /*0x*/ u8 normalFishPole; // 00 = don't have
+    /*0x*/ u8 superFishPole;
+    /*0x*/ u8 ultraFishPole;
+    /*0x*/ u8 chopTutorial;
+    /*0x*/ u8 mysteriouStone;
+    /*0x*/ u8 summonStone;
+    /*0x*/ u8 lotteryHouseBill;
+    /*0x*/ u8 lostCat;
+    /*0x*/ u8 craftEntrust1;
+    /*0x*/ u8 craftEntrust2;
+    /*0x*/ u8 craftEntrust3;
+    /*0x*/ u8 craftEntrust4;
+    /*0x*/ u8 craftEntrust5;
+    /*0x*/ u8 craftEntrust6;
+    /*0x*/ u8 craftEntrust7;
+    /*0x*/ u8 craftEntrust8;
+    /*0x*/ u8 craftEntrust9;
+    /*0x*/ u8 jetsEntrust;
+    /*0x*/ u8 zaksEntrst;
+    /*0x*/ u8 edisEntrust;
+    /*0x*/ u8 entrustFail1;
+    /*0x*/ u8 entrustFail2;
+    /*0x*/ u8 entrustFail3;
+    /*0x*/ u8 shifusWeapon;
+    /*0x*/ u8 mightBook1;
+    /*0x*/ u8 mightBook2;
+    /*0x*/ u8 mightBook3;
+    /*0x*/ u8 mightBook4;
+    /*0x*/ u8 mightBook5;
+    /*0x*/ u8 craftMightBook;
+    /*0x*/ u8 mightBook6; // looks wierd. maybe need a fix
+    /*0x*/ u8 auditionMachine;
+    /*0x*/ u8 waterDragonScale;
+    /*0x*/ u8 inviteToTheGreatWar;
+    /*0x*/ u8 watchingWarInvite;
+    /*0x*/ u8 ex1Invite;
+    /*0x*/ u8 ex2Invite;
+    /*0x*/ u8 jvhuaChaInStory;
+    /*0x*/ u8 luggageToSend;
+    /*0x*/ u8 goldFaction99Entrust;
+    /*0x*/ u8 zaksHammer;
+    /*0x*/ u8 jetEntrustFail;
+    /*0x*/ u8 zaksEntrustFail; // where's edi?
+    /*0x*/ u8 filler53D;
+    /*0x*/ u8 filler53E;
+    /*0x*/ u8 filler53F;
+    /*0x*/ u8 filler540[0x24];
+    /*0x*/ u8 weaponDex[0x1E]; // This is a bit field. 1 = dex lighted, 0 = not lighted. 
+    /*0x*/ u8 filler582; // wierd
```

Comment:
```suggestion
    /*0x*/ u8 filler582; // weird
```
and why?

Reply:
Oh, I don't know why this place need a struct align. ^_^